sentry-android-core/src/main/java/io/sentry/android/core/AndroidTransportGate.java

```diff
@@ -18,7 +19,11 @@
 
   @Override
   public boolean isSendingAllowed() {
-    Boolean connected = ConnectivityChecker.isConnected(context, loger);
+    return isConnected(ConnectivityChecker.isConnected(context, loger));
```

Comment:
I realized this change didn't introduce it but there's a typo on `loger`

Reply:
yep, fixing it, thx